src/UserInfoWidget.cpp

```diff
@@ -216,4 +216,17 @@ UserInfoWidget::paintEvent(QPaintEvent *event)
         opt.init(this);
         QPainter p(this);
         style()->drawPrimitive(QStyle::PE_Widget, &opt, &p, this);
+  
+        Painter p(this);
```

Comment:
You forgot to include this, so it fails to compile.

Reply:
Included #include "ui/Painter.h